diagnostic.md

```diff
@@ -7,36 +7,38 @@ What command do we use to do that? Use the **correct** URL from your fork of
 this repository in your answer.
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+git clone https://github.com/ggwilliams/git-basics-diagnostic.git
 ```
 
 2.What do you do after cloning a repository, before you start making any
 changes/additions?
 
-<!-- Remove this comment and place your answer here. -->
+git status
+atom https://github.com/ggwilliams/git-basics-diagnostic.git
 
 3.What command do we use to create a new branch? Name this branch `response`
     in your answer. Then, how do we switch to that branch?
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+git checkout -b response
+git checkout response
```

Comment:
Correct.  The long way is "git branch response" followed by "git checkout response".  The short way, which you've answered with, is "git checkout -b response".  The second line, "git checkout response" is unnecessary since "git checkout -b <branch>" automatically puts you in that branch.